### PR TITLE
export "computed" from "sinuous" as well

### PR DESCRIPTION
**Pull request by rozek**
First of all: thank you very much for this wonderful package!

Is there a reason, why "sinuous" re-exports "observable" only - and not "computed"? In my applications, reactive values derived from other reactive ones play an important role - for that reason, I would like to see "computed" exported as well.

With greetings from Germany,

Andreas Rozek

### CODE REVIEW

**Comment by luwes** [comment]
thanks for the contribution! will look at this soon

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@ module
 
 # Local Netlify folder
 .netlify
+
+package-lock.json
+yarn.lock
+
```
Comment (luwes) [inline .gitignore:27]:
```suggestion
```

I prefer adding lock files to git

File: src/src/index.js [hunk 2]
```diff
@@ -39,4 +40,4 @@ export const html = (...args) => htm.apply(h, args);
 // `export const svg = htm.bind(hs)` is not tree-shakeable!
 export const svg = (...args) => htm.apply(hs, args);
 
-export { api, o, observable };
+export { api, o, observable, computed };
```
Comment (luwes) [inline src/src/index.js:43]: 👍

File: .gitignore (hunk 1 above)
Comment (rozek) [reply .gitignore]: I've reverted `.gitignore` and re-committed `package-lock.json` and `yarn.lock` as you wanted

**Pull request closed without merging**